src/fu-tool.c

```diff
@@ -1166,6 +1166,32 @@ fu_util_firmware_builder (FuUtilPrivate *priv, gchar **values, GError **error)
 	return fu_common_set_contents_bytes (values[1], firmware_blob, error);
 }
 
+static gboolean
+fu_util_self_sign (FuUtilPrivate *priv, gchar **values, GError **error)
```

Comment:
Need the associated bash completion for this one now

Reply:
Fixed. I wonder if there's a way to fail to build/test/CI if we forget these? I know I'm really bad at remembering to update them...

Reply:
Commands and flags are added (relatively) infrequently, I don't think it's likely worth the trouble of trying to come up with something like that.